changelogs/fragments/1681-add_passwordstore_yaml_support.yaml

```
@@ -0,0 +1,4 @@
+minor_changes:
+  - plugin.lookup.passwordstore - Add support to parse a passwordstore entry as YAML, skipping the first line (which by convention only contains the password and nothing else).
+bugfixes:
+  - plugin.lookup.passwordstore - add support for YAML as a password entry, as stated in the documentation (https://github.com/ansible-collections/community.general/issues/1673)
```

```suggestion
  - passwordstore lookup plugin - add support for YAML as a password entry, as stated in the documentation (https://github.com/ansible-collections/community.general/issues/1673).
```
It would be better to have only one entry (I think `minor_changes` is fine).